Add support for LaTeX math expressions

anschmieg opened the pull request:
**Issue Description**
Currently, the HTML output displays e.g. the example entry
```
Built an app to compute the similarity of all methods
in a codebase, reducing the time from $$\mathcal{O}(n^2)$$
to $$\mathcal{O}(n \log n)$$
```
as a literal, without rendering the math expression. 

**Suggested change**
By adding Mathjax to the document via the Cloudflare CDN, LaTeX and LaTeX math expressions get parsed correctly if the CDN is reachable. 

--- 

I hope I didn't screw up the branches or the commit history again... still working those out 😄

anschmieg commented:
Since there is still an AssertionError, I assume the HTML output doesn't match the expected result because of the added tag?

sinaatalay commented:
Thank you, I haven't thought about this before! I will merge this. Could you use [KaTeX](https://katex.org) instead? It's much faster than MathJax and it's what we use on [docs.rendercv.com](https://docs.rendercv.com).

anschmieg commented:
> I haven't thought about this before

I guess it is a pretty niche use case to need math expressions in your CV. Considering that, it might make sense to lazy-load the lib to avoid unnecessary bloat for everyone else. However, I am not aware of that being possible without using another library, which would kinda defeat the purpose. Thoughts?

> Could you use [KaTeX](https://katex.org) instead? 

Cool, I didn't know that one! I suppose it should be a drop-in replacement... I would still embed it as a JsDeliver reference though, for simplicity's sake. Unless you have a better idea. Shall I update the PR?

sinaatalay commented:
KaTeX is very lightweight, so there is no problem at all with including it in the HTML. If you could update this PR for KaTeX, it will be ready to be merged.

anschmieg commented:
_"I am once again asking for your technical support"_

The test I added in `[5d7e98f](https://github.com/sinaatalay/rendercv/pull/200/commits/5d7e98f621cfb184dcc316763cb342d34f630ca8)` appears to have been a legacy test case. I reverted the change, both the feature and the tests should be fine now. I wouldn't mind a review though.

sinaatalay commented:
Thank you! It was not working but the fix was quick ([details](https://github.com/rendercv/rendercv/pull/200/commits/3142fb6b50362c8890bdc864ada3147d8940918f)).
